pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -325,6 +332,26 @@ func (ci *CloudInfo) getNetworks() ([]string, error) {
 	return networkIDs, nil
 }
 
+// getSecurityGroups returns all the security group IDs available on the cloud.
+func (ci *CloudInfo) getSecurityGroups() ([]string, error) {
+	pages, err := groups.List(ci.clients.networkClient, groups.ListOpts{}).AllPages()
```

Comment:
Change in ln 317 is replacing networks.ListOpts{} with nil.  Do we want to pass in groups.ListOpts{} or nil here?

Reply:
groups.List doesn’t accept nil

Reply:
I have fixed the git history to make the two changes actually atomic, so that this change is a bit more clear